packages/soundfonts/fontloader.mjs

```diff
@@ -1,6 +1,7 @@
-import { noteToMidi, freqToMidi } from '@strudel.cycles/core';
+import { noteToMidi, freqToMidi, _mod } from '@strudel.cycles/core';
 import { getAudioContext, registerSound, getEnvelope } from '@strudel.cycles/webaudio';
 import gm from './gm.mjs';
+import { getSoundIndex } from '../superdough/util.mjs';
```

Comment:
iirc relative imports from another package might break stuff. to be safe, this should either be importing from `@strudel.cycles/superdough` (and make sure it's exported)

Reply:
I couldn't get that import to work, so I just added the function to the utils in core